guides/v2.1/extension-dev-guide/code-generation.md

```diff
@@ -1,15 +1,12 @@
 ---
 group: extension-dev-guide
-subgroup: 99_Module Development
 title: Code generation
-menu_title: Code generation
-menu_order: 8
 version: 2.1
 github_link: extension-dev-guide/code-generation.md
 ---
 
-<h2 id="codegen-over">Overview of code generation</h2>
-The Magento application generates code to create non-existent classes. As an example, look at the <a href="{{ site.mage2000url }}/app/code/Magento/Customer/Model/ResourceModel/AddressRepository.php" target="_blank">\Magento\Customer\Model\Resource\AddressRepository</a> constructor. A snippet follows:
+## Overview of code generation {#codegen-over}
+The Magento application generates code to create non-existent classes. As an example, look at the [\\Magento\\Customer\\Model\\Resource\\AddressRepository] constructor. A snippet follows:
```

Comment:
Why are there two \\ ? Should be [Magento/Customer/Model/Resource/AddressRepository]

Reply:
Fixed